webdriver/tests/bidi/network/add_intercept/add_intercept.py

```diff
@@ -58,6 +58,58 @@ async def test_other_context(
     await fetch(text_url, context=other_context)
 
 
+@pytest.mark.asyncio
+async def test_other_context_with_event_subscription(
```

Comment:
Let's extract this in a `contexts.py` file similar to other test.

Reply:
Could you elaborate? Which other test? this test is very similar to tests that are already in this file.

Reply:
In general we try to create a dedicated test file for each parameter we are testing. 

When I added the initial tests for this command I could not really decide if I should put them in `phases.py` or `url_patterns.py` because they are really not testing one of the 2 parameters in particular. 

But here since the test is really about `contexts`, it should be fine to create a new test file for it.

Reply:
Moved both tests to contexts.py